gnatsd/plan.sh

```diff
@@ -26,11 +24,12 @@ do_build() {
   cp -r conf ./src/github.com/nats-io/gnatsd
   cp -r util ./src/github.com/nats-io/gnatsd
   cp -r vendor ./src/github.com/nats-io/gnatsd
-
   go build
 }
 
 
 do_install() {
+  mkdir -p "${pkg_prefix}/bin"
   cp -R . "${pkg_prefix}"
+  ln -s "${pkg_prefix}/${pkg_name}-${pkg_version}" "${pkg_prefix}/bin/${pkg_name}-${pkg_version}"
```

Comment:
During the installation step we typically move files to the locations we want them with `install -D`. Here's a good example: https://github.com/habitat-sh/core-plans/blob/master/busybox/plan.sh#L44-L51